homeassistant/components/media_player/mpd.py

```diff
@@ -142,12 +142,15 @@ def media_duration(self):
     def media_title(self):
         """ Title of current playing media. """
         name = self.currentsong.get('name', None)
-        title = self.currentsong['title']
+        title = self.currentsong.get('title', None)
 
         if name is None:
             return title
         else:
-            return '{}: {}'.format(name, title)
+            if title is None:
```

Comment:
Instead of
```
else:
    if title is None:
```

rewrite both if-blocks to be just `if X, elsif Y, elsif Z, else`